test/unit/logic/utils.spec.ts

```diff
@@ -0,0 +1,226 @@
+import { Entity, Rarity, WearableCategory } from '@dcl/schemas'
+import { buildTrimmedEntity } from '../../../src/logic/utils'
+import { ItemType } from '../../../src/types'
+
+describe('utils', function () {
+  describe('buildTrimmedEntity', function () {
+    const createMockEntity = (overrides?: Partial<Entity>): Entity => {
+      return {
+        id: 'test-entity-id',
+        type: 'wearable',
+        pointers: ['urn:decentraland:matic:collections-v2:0xtest:0'],
+        timestamp: Date.now(),
+        content: [
+          {
+            file: 'thumbnail.png',
+            hash: 'QmThumbnailHash123'
+          },
+          {
+            file: 'model.glb',
+            hash: 'QmModelHash456'
+          }
+        ],
+        metadata: {
+          id: 'metadata-id',
+          name: 'Test Wearable',
+          description: 'A test wearable',
+          rarity: Rarity.COMMON,
+          thumbnail: 'thumbnail.png',
+          data: {
+            category: WearableCategory.HAT,
+            representations: [
+              {
+                bodyShapes: ['urn:decentraland:off-chain:base-avatars:BaseMale'],
+                mainFile: 'model.glb',
+                contents: ['model.glb'],
+                overrideHides: [],
+                overrideReplaces: []
+              }
+            ],
+            tags: []
+          }
+        },
+        ...overrides
+      } as Entity
+    }
+
+    describe('isSmart flag', function () {
+      let entity: Entity
+
+      beforeEach(function () {
+        entity = createMockEntity()
+      })
+
+      it('should return isSmart as true when itemType is SMART_WEARABLE_V1', function () {
```

Comment:
Would you mind refactoring these tests so they follow our testing standards better?